### ISSUE TITLE
SEC-26: Fix SSL/TLS issues for Java 11

### ISSUE THREAD

**Issue by arkarkala** [OpenLiberty/open-liberty#5074]
As part of the Java 11 support, the following items need to be considered and fixed as needed.

TLS 1.3 support
Use PKCS12 keystore type by default instead of JKS type
Other things that come up as part of Java 11 testing.

**Comment by atosak** [comment]
Ship readiness checklist is being tracked here:  https://github.com/OpenLiberty/open-liberty/issues/5452

**Comment by marikaj123** [comment]
Translation id needed.

**Comment by marikaj123** [comment]
According to Ajay, the translated files were already included in the release.

**Comment by gscottj** [comment]
Accessibility Approval Comment:
Based on the description provided, it appears that this feature does not have any end-user user interfaces that require accessibility compliance testing. If this assumption is not correct, then let me know that required accessibility verification testing of the UI will be performed before it is shipped and what are the results from those tests.

**Comment by sabolo** [comment]
I am doing a quick run through of features in advance of the Feature FAT approvals.  At present I am unable to find any links to a WAD, either from this Epic of in the AHA feature.  Please can you ensure there are links to the WAD in the top level description of this Epic.  Many thanks

**Comment by sabolo** [comment]
I am looking into this feature for providing the Feature FAT approval.  At present I cannot find the Feature Test Summary issue that should be under this Epic.  Please can you add the issue as expected... If unsure on the process you can find it here:  https://github.ibm.com/was-liberty/WS-CD-Open/wiki/Feature-Review.  Many thanks

**Comment by marikaj123** [comment]
@arkarkala - Do you have all the translated files needed for this feature?

**Comment by arkarkala** [comment]
This feature is only handling supporting TLS 1.3 in Java 11. This should not need any STE since it is like any other protocol that is currently supported. Will work with STE folks to confirm.

**Comment by arkarkala** [comment]
There are no new fvt test cases required for this feature since the current ssl config handles tls1.3 protocol automatically. SVT will be adding some tests just to test tls 1.3.

**Comment by arkarkala** [comment]
@marikaj123 - no translation files for this feature. However, the other feature that was included in this earlier, which has been now split into sec-41, had translated files but they have been in master since November.

**Comment by arkarkala** [comment]
@donbourne - the existing serviceability messages will also handle TLS 1.3 protocol, just like the other older protocols.

**Comment by arkarkala** [comment]
@sabolo , the Feature Test Summary should be completed here - https://github.com/OpenLiberty/open-liberty/issues/6272 . Let me know if anything needs else needs to be done. Thanks,

**Comment by sabolo** [comment]
At present I am happy with the summary etc in relation to Java 11 OpenJ9.  This feature though is to cover all Java 11 versions including hotspot and we have not yet seen results from hotspot.  As such I don't believe we can close this out.  If you have other Epics covering Java 11 hotspot please can you point me at them, but in the meantime, similar to other Java 11 features, I will be holding this open pending hotspot support.

**Comment by marikaj123** [comment]
According to ANDREW (ANDY) GUIBERT:
hopefully just 1 PR to enable TLSv1.3 for Hotspot, and any necessary test updates associated with that 
but right now we are waiting on the next Java 11 fix pack to determine.

**Comment by marikaj123** [comment]
@arkarkala - Please continue to work with @atosak and the focal point owners @chirp1 and @sabolo to  obtain the rest of the focal point owners and get the feature closed.  My understanding, the feature was delivered in 19001.

**Comment by atosak** [comment]
@marikaj123 For this feature and some other Java 11 related features, The test team have not provided approval simply because we also need to cover Hotspot. What we enabled in 19001 is Liberty support on Open J9. We are waiting on Hotspot 11.0.3 to be released, which should contain the fixes we need. It should be released in the next month or two, at which point we will be un-blocked.

**Comment by marikaj123** [comment]
@atosak - Thank you for being on top of it!

**Comment by arkarkala** [comment]
This has been already delivered so closing.
